libcxx/test/std/containers/container.adaptors/flat.set/flat.set.capacity/empty.verify.cpp

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I think this should be libc++ specific. You can also put this in `libcxx/test/libcxx/diagnostics/flat_set.nodiscard.verify.cpp` since that's where we have all of these tests (perhaps also check what you did for `flat_map`, since I think it wasn't done consistently).